apps/dotcom/client/src/tla/utils/app-ui-events.tsx

```diff
@@ -8,6 +9,8 @@ export type TLAppUiEventSource =
 	| 'file-rename-dialog'
 	| 'file-menu'
 	| 'file-share-menu'
+	| 'file-header'
+	| 'anon-landing-page'
```

Comment:
Not sure about the source here, do we have an established name for this?

Reply:
Feels ok 🤷‍♂️ Another option might be `signed-out-page`?